web/apps/labelstudio/src/themes/default/colors.scss

```diff
@@ -1,21 +1,17 @@
 :root {
-  --black-raw: 0, 0, 0;
-  --white-raw: 255, 255, 255;
-  --accent_color-raw: 86, 111, 207;
-  --danger_color-raw: 221, 0, 0;
+  --black-raw: 0 0 0;
+  --white-raw: 255 255 255;
+  --accent_color-raw: 86 111 207;
+  --danger_color-raw: 221 0 0;
   --black: rgb(var(--black-raw));
   --white: rgb(var(--white-raw));
-  --accent_color: rgb(var(--accent_color-raw));
-  --danger_color: rgb(var(--danger_color-raw));
-
-  --black_2:  rgba(var(--black-raw), 0.02);
-  --black_4:  rgba(var(--black-raw), 0.04);
-  --black_5:  rgba(var(--black-raw), 0.05);
-  --black_10: rgba(var(--black-raw), 0.1);
-  --black_15: rgba(var(--black-raw), 0.15);
-  --black_20: rgba(var(--black-raw), 0.20);
-  --black_40: rgba(var(--black-raw), 0.40);
-
+  --black_2:  rgb(var(--black-raw) / 2%);
```

Comment:
Notice I remove -accent_color: and --danger_color they are repeated down the file, I assume the right values is the bottom one.

Reply:
fair point but if we clear out the tome definitions we should also clear out the `-raw` variables since they dont seem to be used now
`--accent_color-raw: 86 111 207;` and `--danger_color-raw: 221 0 0;`

Reply:
this is also a non-issue for `--accent_color` because the colors evaluate to the same value but `--danger-color-raw` and `--red_7` dont evaluate to the exact same RGB value when we convert red_7 over (although its very close)